src/MahApps.Metro/Styles/Controls.Calendar.xaml

```diff
@@ -114,8 +115,8 @@
                         <!--  IsToday and IsTodayHighlighted  -->
                         <MultiDataTrigger>
                             <MultiDataTrigger.Conditions>
-                                <Condition Binding="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type Calendar}}, Path=IsTodayHighlighted}" Value="True" />
-                                <Condition Binding="{Binding RelativeSource={RelativeSource Self}, Path=IsToday}" Value="True" />
+                                <Condition Binding="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type Calendar}}, Path=IsTodayHighlighted, FallbackValue=False}" Value="True" />
```

Comment:
The FallbackValue should be True, because the default of the property is also True

Reply:
agreed